omod/src/main/java/org/openmrs/module/ugandaemr/fragment/controller/PharmacyQueueListFragmentController.java

```diff
@@ -39,7 +41,10 @@ public void controller(@SpringBean FragmentModel pageModel, UiSessionContext uiS
         SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy");
         String dateStr = sdf.format(new Date());
         List<String> list = new ArrayList();
-        list.add(PHARMACY_LOCATION_UUID);
+
+        list.addAll(Context.getLocationService().getLocationsByTag(Context.getLocationService().getLocationTagByUuid("fe7c970f-2aba-11ed-ba4a-507b9dea1806")).stream().map(Location::getUuid).collect(Collectors.toList()));
```

Comment:
@jwasswa  could we have the uuids put as constants instead as inline code references